spec/controllers/adoptions_controller_spec.rb

```diff
@@ -0,0 +1,363 @@
+        end
+
+        it 'se verifica que se actualice el animal' do
+          post :create, adoption: params, format: 'json'
+          expect(animal.reload.adopted).to eq(true)
+        end
+
+        it 'se verifica el evento creado' do
+          post :create, adoption: params, format: 'json'
+          new_event = Event.last
+          expect(new_event.animal_id).to eq(params[:animal_id])
+          expect(new_event.description).to eq("El animal fue adoptado por #{adopter.first_name} #{adopter.last_name}.")
+          expect("#{new_event.date}").to eq(params[:date])
+        end
+      end
+    end
+
+    context 'no exitoso' do
+      context 'por un usuario con permiso por defecto' do
+        before(:each) do
+          request.headers['X-USER-TOKEN'] = user2.authentication_token
+        end
+
+        it 'devuelve codigo 403' do
+          post :create, adoption: params, format: 'json'
+          expect(response.status).to eq(403)
+        end
+
+        it 'la cantidad de adopciones no aumenta' do
+          expect { post :create, adoption: params, format: 'json' }.not_to change { Adoption.count }
+        end
+
+        it 'devuelve mensaje de error' do
+          post :create, adoption: params, format: 'json'
+          expect(parse_response(response)['error']).to eq('Faltan permisos')
+        end
+      end
+
+      context 'por un usuario con permiso de editar adoptantes' do
+        before(:each) do
+          request.headers['X-USER-TOKEN'] = user3.authentication_token
+        end
+
+        it 'devuelve codigo 403' do
+          post :create, adoption: params, format: 'json'
+          expect(response.status).to eq(403)
+        end
+
+        it 'la cantidad de adopciones no aumenta' do
+          expect { post :create, adoption: params, format: 'json' }.not_to change { Adoption.count }
+        end
+
+        it 'devuelve mensaje de error' do
+          post :create, adoption: params, format: 'json'
+          expect(parse_response(response)['error']).to eq('Faltan permisos')
+        end
+      end
+
+      context 'con un adoptante en blacklist' do
+        before(:each) do
+          request.headers['X-USER-TOKEN'] = user.authentication_token
+        end
+
+        let(:params) { { adopter_id: adopter2.id, animal_id: animal.id, date: '2016-11-01' } }
+
+        it 'devuelve codigo 422' do
+          post :create, adoption: params, format: 'json'
+          expect(response.status).to eq(422)
+        end
+
+        it 'la cantidad de adopciones no aumenta' do
+          expect { post :create, adoption: params, format: 'json' }.not_to change { Adoption.count }
+        end
+
+        it 'devuelve mensaje de error' do
+          post :create, adoption: params, format: 'json'
+          expect(parse_response(response)['error']['adopter_id']).to eq(['El adoptante está en la lista negra.'])
+        end
+
+        it 'se verifica que no se haya creado la adopcion' do
```

Comment:
Me parece que este test está de mas, ya se controla que no se haya creado la adopcion en `it 'la cantidad de adopciones no aumenta' do ...`